pingouin/nonparametric.py

```diff
@@ -281,14 +288,33 @@ def mwu(x, y, alternative='two-sided', **kwargs):
 
     # Effect size 2: rank biserial correlation (Wendt 1972)
     rbc = 1 - (2 * uval) / diff.size  # diff.size = x.size * y.size
-
+    
+    # Confidence interval for the (difference in) medians 
+    # Campbell and Gardner 2000
+    if alternative == "two-sided":
+        alpha = 1.0 - confidence
+        conf = 1.0 - alpha / 2  # 0.975
+    else:
+        conf = confidence
+    N = scipy.stats.norm.ppf(conf) 
+    ct1, ct2 = len(x),len(y) # count samples 
+    diffs = sorted([i-j for i in x for j in y]) # get ct1xct2 difference
```

Comment:
@kschuerholt could we use a numpy function / numpy broadcasting here to avoid the nested for loop?

Reply:
Sure, that's easy enough. I'll add it in a new commit promptly.